product.py

```diff
@@ -280,6 +281,9 @@ def fetchIfaceInfoFromNetworkdbAsDict(bridge, iface=None):
                     results['net-admin-configuration'].addIPv6(NetInterface.DHCP)
                 elif protov6 == 'autoconf':
                     results['net-admin-configuration'].addIPv6(NetInterface.Autoconf)
+            else:
+                logger.log("Failed to find netword.db on filesystem")
```

Comment:
I see a typo here, what about:
```suggestion
                logger.log("Failed to find " + self.join_state_path(constants.NETWORK_DB))
```